.github/workflows/frontend.yml

```diff
@@ -88,9 +88,14 @@ jobs:
       uses: ./.github/actions/prepare-backend
       with:
         m2-cache-key: 'cljs'
-    - run: yarn run test-unit --coverage --silent
-      name: Run frontend unit tests
+    - name: Run frontend unit tests
+      if: github.ref == 'refs/heads/master'
+      run: yarn run test-unit --coverage --silent
+    - name: Run frontend unit tests and collect the code coverage information
+      if: github.ref != 'refs/heads/master'
```

Comment:
We can maybe add a release branch to the mix, but it shouldn't really matter because anything that goes into the release branch first lands into master.